pkg/destroy/openstack/openstack.go

```diff
@@ -653,6 +653,57 @@ func getRouterByPort(client *gophercloud.ServiceClient, allPorts []ports.Port) (
 	return "", nil
 }
 
+func deleteLeftoverLoadBalancers(opts *clientconfig.ClientOpts, logger logrus.FieldLogger, networkID string) (bool, error) {
+	conn, err := clientconfig.NewServiceClient("load-balancer", opts)
+	if err != nil {
+		// Ignore the error if Octavia is not available for the cloud
+		var gerr *gophercloud.ErrEndpointNotFound
```

Comment:
here you use it as a pointer type, and below in the similar clause, as a refular type. This doesn't look consistent?